srcpkgs/spicetify-cli/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'spicetify-cli'
+pkgname=spicetify-cli
+version=2.9.9
+revision=1
+archs="x86_64"
```

Comment:
why?

Reply:
If you are talking about `archs`, I have explained that in the description. 

> NOTE: as the Spotify package is locked for x86_64-glibc i locked this package for that too, though it does offer aarch64 pre-built binaries.